src/types/signable-token.ts

```diff
@@ -0,0 +1,55 @@
+import { SignableToken } from '../api';
+
+export interface ERC20Token {
```

Comment:
Nice work @zatlite ! This is so much cleaner now. One suggestion is to rename the file as `signable-token.ts` doesn't seem to make sense anymore. Maybe `tokens.ts`?

Reply:
Good idea. [fc419f7](https://github.com/immutable/imx-core-sdk/pull/149/commits/fc419f7bcaa9ab23ddeef88c47a8f7389b7f90da)